### PR TITLE
msieve: fix build on macOS 12 arm64

### PR DESCRIPTION
**Pull request by catap**
###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 12.0.1 21A559 arm64
Xcode 13.1 13A1030d

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [ ] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

**Review by mascguy — APPROVED** [review]

**Comment by ryandesign** [comment]
@mascguy You should allow portfile maintainers an opportunity (72 hours) to respond to proposed changes to their ports. Looks like this PR was merged only 16 minutes after it was opened.

**Comment by ryandesign** [comment]
(And what makes me point it out is not that specifically, but the fact that the proposed changes have some issues that might have been noticed and fixed before merging, had the PR been left open longer.)

**Comment by catap** [comment]
@ryandesign which issue?

**Comment by mascguy** [comment]
> @mascguy You should allow portfile maintainers an opportunity (72 hours) to respond to proposed changes to their ports. Looks like this PR was merged only 16 minutes after it was opened.

Agreed, and lesson learned.

**Pull request merged**
